src/rewards/Forms/FormCardLinkButton.tsx

```diff
@@ -21,29 +21,48 @@ const FormCardLinkButton = ({ platform }: Props) => {
 
   const { userSubmission, isValidating } = useUserFormSubmission(form)
 
-  if (isAdmin && !!userSubmission)
+  if (isFormsValidating || isValidating) return <RewardCardButton isLoading />
+  if (!form) return <RewardCardButton isDisabled>Form not found</RewardCardButton>
+
+  if (!userSubmission)
```

Comment:
I'm not sure if we should just move the `isAdmin` block before this one (that way the admin won't be able to fill the form easily?), or we should actually move the "view responses" link to the reward card menu. What's your opinion on this one?